code/modules/mob/living/simple_animal/hostile/mining_mobs/ice demon.dm

```diff
@@ -85,3 +85,17 @@
 	if(prob(5))
 		new /obj/item/raw_anomaly_core/bluespace(loc)
 	return ..()
+
+
+//SKYRAT CHANGE, NERFED ICE DEMON FOR MINING
+/mob/living/simple_animal/hostile/asteroid/ice_demon/mining
```

Comment:
Change the drop list; they're an easier version of the demonic watcher so their drops should be worse-ish